apps/cms-server/views/partials/footer.html

```diff
@@ -23,8 +21,61 @@ <h2>{{column.title}}</h2>
     </div>
   </div>
 </footer>
-{% endif %}
 <div class="body-background" style="display: none;"></div>
 <div id="openstad-toast" class="toast-container initial-width">
+</div> -->
+
+<div
+  id="footer-container"
+  data-content='[
+        {
+          "title": "Over deze site",
+          "items": [
+            {
+              "url": "#",
+              "label": "Privacy"
+            },
+            {
+              "url": "#",
+              "label": "Toegankelijkheid"
+            },
+            {
+              "url": "#",
+              "label": "Cookies"
+            }
+          ]
+        },
+        {
+          "title": "Taal",
+          "items": [
+            {
+              "url": "#",
+              "label": "NL Nederlands"
+            },
+            {
+              "url": "#",
+              "label": "EN English"
+            }
+          ]
+        },
+        {
+          "title": "Contact",
+          "items": [
+            {
+              "url": "#",
+              "label": "Veelgestelde vragen"
+            },
+            {
+              "url": "#",
+              "label": "Hou mij op de hoogte"
+            },
+            {
+              "url": "#",
+              "label": "Neem contact op"
+            }
+          ]
+        }
+      ]'
```

Comment:
Voor nu prima idd, maar zou mooi zijn als je via de Global settings een footer menu op kan bouwen (kon in de oude versie ook namelijk)